internal-dns/tests/basic_test.rs

```diff
@@ -140,10 +140,12 @@ async fn init_client_server(
     {
         let db = db.clone();
         let log = log.clone();
-        let config = config.dns.clone();
+        let dns_config = internal_dns::dns_server::Config {
+            bind_address: format!("127.0.0.1:{}", dns_port),
```

Comment:
Can we use an IPv6 address here? I'd like to keep from adding more IPv4 if possible.

Reply:
Sure, I can update this whole file to be IPv6 if you want - though it is both test-only and basically just the dropshot config type, which I don't think we will ever change to be IPv6-only.

Reply:
Ah, didn't see it's just a test. I don't feel strongly about it then, I'm just hoping to get us uniformly on IPv6 for all Oxide stuff. Up to you!